import.sh

```diff
@@ -2,17 +2,20 @@
 
 set -eu
 
-rm -rf *.c internal/*
+rm -rf internal/*
+find . -type l | xargs rm
```

Comment:
We don't want to delete any symlinks that may exist in the `.git` directory. (not sure if that ever happens "naturally", but in our main repo we put symlinks in `.git/hooks`)